test/overlay.html

```diff
@@ -347,6 +359,58 @@
         });
       });
     });
+
+    describe('frontmost', () => {
+      var dialog;
+      beforeEach(() => {
+        dialog = fixture('in-modeless-dialog');
+      });
+
+      function touchstart(node) {
+        const event = new CustomEvent('touchstart', {bubbles: true, cancelable: true});
+    
+        const nodeRect = node.getBoundingClientRect();
+        const clientX = nodeRect.left;
+        const clientY = nodeRect.top;
+
+        event.touches = event.changedTouches = event.targetTouches = [{clientX, clientY}];
+        node.dispatchEvent(event);
+      }
+
+      it('should not end up behind the dialog overlay on mousedown', (done) => {
+        dialog.opened = true;
+        const dialogOverlay = dialog.$.overlay;
+        const datePicker = dialog.$.overlay.querySelector('vaadin-date-picker');
+        const datePickerOverlay = datePicker.$.overlay;
+    
+        datePicker.opened = true;
+    
+        Polymer.RenderStatus.afterNextRender(datePicker, () => {
```

Comment:
Is this necessary?

Reply:
checked, and it is not necessary.  I removed it.